com.unity.perception/Runtime/GroundTruth/Labelers/CameraLabeler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    [Serializable]
+    public abstract class CameraLabeler
+    {
+        public bool enabled = true;
+        public bool foldout = true;
+
+        internal bool isInitialized { get; private set; }
+
+        protected PerceptionCamera perceptionCamera { get; private set; }
+        protected SensorHandle SensorHandle { get; private set; }
```

Comment:
Good call. Had some discussion with Adriano on the scripting team, and since our code is in UnityEngine we should stick to camelCase property names. Switching to that.